pipelines/optimus/Optimus.wdl

```diff
@@ -222,5 +243,9 @@ workflow Optimus {
 
       # zarr
       Array[File]? zarr_output_files = OptimusZarrConversion.zarr_output_files
+
+      # fastqc
+      Array[Array[File]] fastqc_htmls = fastqc_output_htmls
```

Comment:
Unless you really want `Array[Array[File]]` here it seems like you'd be better off with `flatten(fastqc_output_htmls)`

Reply:
agreed, will do.